packages/ui/src/components/DataStory/dataStoryControls.tsx

```diff
@@ -53,12 +54,12 @@ export function DataStoryControls({
     <ControlButton
       title="Run"
       aria-label="Run"
-      onClick={() => setShowRunModal(true)}
+      onClick={() => setShowRunModal('run')}
```

Comment:
Is there a reason for passing already implied 'run' into this function; why pass true/false?

Reply:
All components displayed in the sidebar are now uniformly managed using `sidebarKey`. Indeed, this change should not affect the logic within `DataStoryControls`, and I have made the necessary adjustments. :thumbsup: